src/main/resources/intentionDescriptions/UnwrapToTryIntention/description.html

```diff
@@ -1,5 +1,6 @@
 <html>
 <body>
-This intention transforms an expression with unwrap to a try operator.
+This intention transforms an expression with <code>.unwrap()</code> to a
+<code>try</code> (<code>?</code>) operator.
```

Comment:
nit: maybe just drop `try` name and replace it with `?`?

For example, [Rust book](https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html#a-shortcut-for-propagating-errors-the--operator) doesn't also use `try` name

Reply:
Hm, surprisingly the Reference calls it simply the [question mark operator](https://doc.rust-lang.org/reference/expressions/operator-expr.html#the-question-mark-operator). Everyone in forums calls it `try`, but seems like it's an unofficial name, unfortunately. I guess the `Try` trait being unstable contributes to that.

Given the lack of official name, just using `?` seems the proper decision.